bin/git-stamp

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+
+init_variables() {
+  COMMAND=${0#*-}
+
+  REPLACE=false
+  unset ID
+  unset MSG
+}
+
+
+usage() {
+  cat << EOF
+usage: git ${COMMAND} [<options>] <id> [<messages>]
+
+Options:
+    -r, --replace    replace all previous stamps with same id
+EOF
+}
+
+
+error() {
+  if [[ -n "$1" ]]; then
+    local msg=$( echo "error: $1" | sed 's/\\n/\\n       /g' )
```

Comment:
Ah that a copy paste from my previous `git reauthor` command
https://github.com/tardypad/git-extras/blob/stamp/bin/git-reauthor#L32
This was for multiple lines error message so that the second line is aligned with the `:` as in
``` 
error: missing target of the rewrite
       use either --old-email option or --all flag
```
It's not needed here, I better remove it then I guess